dowhy/causal_identifier/id_identifier.py

```diff
@@ -87,8 +88,8 @@ def __str__(self):
             return string
 
 
-class IDIdentifier(CausalIdentifier):
-    def __init__(self, graph, estimand_type, method_name="default", proceed_when_unidentifiable=None):
+class IDIdentifier:
+    def __init__(self, estimand_type=EstimandType.NONPARAMETRIC_ATE):
```

Comment:
It looks like this parameter is never used. Can it be removed?

Reply:
Yup, removing in next commit